lib/hotwire/livereload/engine.rb

```diff
@@ -15,10 +15,8 @@ class Engine < ::Rails::Engine
         #{root}/app/helpers
       )
 
-      initializer "hotwire_livereload.assets" do
-        if Rails.application.config.respond_to?(:assets)
-          Rails.application.config.assets.precompile += %w( hotwire-livereload.js )
-        end
+      initializer 'hotwire_livereload.assets.precompile' do |app|
+        app.config.assets.precompile += %w( hotwire-livereload.js )
```

Comment:
I think we still need to check if config responds to `assets` as the app might not use sprockets and use https://github.com/rails/propshaft instead.

Reply:
yep! Great call, I've updated the PR 🙌

this is something I've got to add to my gem too 😆  noticed it on one of the projects